heartbeat/monitors/active/http/http.go

```diff
@@ -57,12 +57,12 @@ func create(
 
 	jobs := make([]monitors.Job, len(config.URLs))
 
-	if config.ProxyURL != "" {
-		transport, err := newRoundTripper(&config, tls)
-		if err != nil {
-			return nil, err
-		}
+	transport, err := newRoundTripper(&config, tls)
+	if err != nil {
+		return nil, err
+	}
 
+	if transport.Proxy != nil {
```

Comment:
This significantly changes the monitor behavior, as transport.Proxy will actually never be nil... meaning `newHTTPMonitorHostJob` will be used to create the monitor job and not `newHTTPMonitorIPsJob`, although I dont quite understand why there are two methods (the implementation looks quite different).